configure.ac

```diff
@@ -1011,6 +1017,13 @@ AS_IF([test "x$with_arkode" != "x" && test "x$with_arkode" != "xno"], [
     #include <arkode/arkode_bbdpre.h>
     extern int arkode_bbd_rhs(ARKODEINT, double, N_Vector, N_Vector, void *);
   ], [ARKBBDPrecInit(nullptr, 0, 0, 0, 0, 0, 0, arkode_bbd_rhs, nullptr);])
+  has_some_sundials="yes"
+])
+
+# Set a flag to enable compilation of hybrid OpenMP-MPI NVector
+AS_IF([test "x$has_some_sundials" != "no" && test "x$enable_sundials_openmp" != "x" && test "x$enable_sundials_openmp" != "xno"],[
```

Comment:
We're in complete control of `$has_some_sundials`, so this could just be `$has_some_sundials == "no"`